be/src/vec/data_types/data_type_factory.cpp

```diff
@@ -311,13 +311,13 @@ DataTypePtr DataTypeFactory::create_data_type(const TypeIndex& type_index, bool
     case TypeIndex::Decimal64:
         nested = std::make_shared<DataTypeDecimal<Decimal64>>(BeConsts::MAX_DECIMAL64_PRECISION, 0);
         break;
-    case TypeIndex::Decimal128:
-        nested = std::make_shared<DataTypeDecimal<Decimal128>>(BeConsts::MAX_DECIMAL128_PRECISION,
-                                                               0);
+    case TypeIndex::Decimal128V2:
+        nested = std::make_shared<DataTypeDecimal<Decimal128V2>>(BeConsts::MAX_DECIMAL128_PRECISION,
+                                                                 0);
```

Comment:
BeConsts::MAX_DECIMAL128_PRECISION maybe should rename this variable.
And also I think line314 is wrong , should use MAX_DECIMALV2_PRECISION?